src/handlers/sendEmailRequestToNotify.ts

```diff
@@ -0,0 +1,41 @@
+import { APIGatewayProxyEvent } from 'aws-lambda'
+import { NotifyClient } from 'notifications-node-client'
+import { retrieveNotifySecrets } from '../secrets/retrieveNotifySecrets'
+import { CustomAxiosResponse } from '../types/notify/axiosResponse'
+import { tryParseJSON } from '../utils/helpers'
+
+// event type is a placeholder
+export const handler = async (event: APIGatewayProxyEvent) => {
+  if (!event.body) {
+    console.log('Could not find event body. An email has not been sent')
```

Comment:
I think this should probably be `console.error` because if this happens it indicates something wrong in the caller, so we'll want to know about it. I'm assuming at some point we'll be able to filter errors in our logs, and do things like get notifications when we get a lot of them.